var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -45,6 +46,16 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
         description="Build C API compatible with ScaLAPACK",
     )
 
+    variant("mpi-gpu-aware", default=False, when="@0.5.0:", description="Use GPU-aware MPI.")
+    conflicts("+mpi-gpu-aware", when="~cuda ~rocm", msg="GPU-aware MPI requires +cuda or +rocm")
+
+    variant(
+        "mpi-gpu-force-contiguous",
+        default=True,
+        when="@0.5.0: +mpi-gpu-aware",
+        description="Force communication buffers to be contiguous before communicating.",
```

Comment:
I was sure I'd added the word "GPU" in the description here, but it seems not (i.e. this matches the DLA-Future definition). However, maybe it'd be good to say:
```suggestion
        description="Force GPU communication buffers to be contiguous before communicating.",
```

Reply:
also consider `-` -> `_`, cause some people complain that `+mpi-gpu` looks too much like `+mpi ~gpu` etc